augur/tasks/git/dependency_tasks/dependency_util/kotlin_deps.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import re
+from pathlib import Path
+
+def get_files(path):
```

Comment:
Does this handle cases, if applicable, where several imports are made on the same line?